src/theme/components/hero.ts

```diff
@@ -0,0 +1,247 @@
+// From the `wrapper` SCSS mixin.
+// TODO: Eventually we may be able to put shared JS style objects
+// into a utils file for other mixins that are shared.
+const wrapperStyles = {
+  marginY: 0,
+  marginX: "auto",
+  maxWidth: "1280px",
+  paddingTop: 0,
+  paddingBottom: 0,
+  paddingRight: 0,
+  paddingLeft: 0,
+  width: "100%",
+};
+// TODO: Is this being used?
+const placeholder = {
+  "+ .placeholder": {
+    flex: "0 1 200px",
+    marginLeft: 12, // --space-xl
```

Comment:
For the spacing, I've been using the values that align with the old CSS vars.  So, in this case, I would use `xl`.  I feel those values are a little more concrete than the numeric values.

Reply:
Are you suggesting using `"xl"` instead of `12`? Internally, they're both the same in `spacing.ts`:

```
// --space-xl = 48px
xl: "3rem",
12: "3rem",
```

Reply:
Hmm I see what you're saying now; it's easier to cognitively think about the name values than the numeric values even though they're the same. Is that right? Yeah, I can move to name values if that's what you mean.

Reply:
Yes, that is exactly right.

Reply:
Updated this.